src/node_crypto.cc

```diff
@@ -3101,20 +3108,26 @@ void CipherBase::Init(const FunctionCallbackInfo<Value>& args) {
   CipherBase* cipher;
   ASSIGN_OR_RETURN_UNWRAP(&cipher, args.Holder());
 
-  CHECK_GE(args.Length(), 2);
+  CHECK_GE(args.Length(), 3);
 
   const node::Utf8Value cipher_type(args.GetIsolate(), args[0]);
   const char* key_buf = Buffer::Data(args[1]);
   ssize_t key_buf_len = Buffer::Length(args[1]);
-  cipher->Init(*cipher_type, key_buf, key_buf_len);
+  CHECK(args[2]->IsInt32());
+  // Don't assign to cipher->auth_tag_len_ directly; the value might not
+  // represent a valid length at this point.
+  int auth_tag_len = args[2]->Int32Value();
```

Comment:
This could/should be `args[2].As<Int32>()->Value()` – `Int32Value()` without a `context` argument is deprecated :) (ditto below)